Gems/Atom/RPI/Code/Include/Atom/RPI.Public/ViewProviderBus.h

```diff
@@ -30,6 +30,7 @@ namespace AZ
             using BusIdType = AZ::EntityId;
 
             virtual ViewPtr GetView() const = 0;
+            virtual ViewPtr GetStereoscopicView(uint32_t viewIndex) const = 0;
```

Comment:
There appears to be a mixture of `uint32_t` and `AZ::u32` throughout this for different APIs for `viewIndex` (see XRRenderingInterface.h).  Is there a reason behind the decision to use `uint32_t`?  I think in general we prefer to use `AZ` typedefs for overall consistency.